Source/WebCore/page/ContextMenuController.cpp

```diff
@@ -536,6 +525,9 @@ void ContextMenuController::contextMenuItemSelected(ContextMenuAction action, co
     case ContextMenuItemTagDictationAlternative:
         frame->editor().applyDictationAlternative(title);
         break;
+    case ContextMenuItemTagShowFonts:
+    case ContextMenuItemTagStyles:
+    case ContextMenuItemTagShowColors:
```

Comment:
Will these now `ASSERT_NOT_REACHED` on other platforms?  Should we wrap them in a `#if PLATFORM(MAC)`?